src/main/java/com/exasol/adapter/dynamodb/mapping/dynamodb/DynamodbToDecimalPropertyToColumnValueExtractor.java

```diff
@@ -0,0 +1,34 @@
+package com.exasol.adapter.dynamodb.mapping.dynamodb;
+
+import java.math.BigDecimal;
+
+import com.exasol.adapter.dynamodb.documentnode.DocumentNode;
+import com.exasol.adapter.dynamodb.documentnode.dynamodb.DynamodbNodeVisitor;
+import com.exasol.adapter.dynamodb.documentnode.dynamodb.DynamodbNumber;
+import com.exasol.adapter.dynamodb.mapping.ToDecimalPropertyToColumnMapping;
+import com.exasol.adapter.dynamodb.mapping.ToDecimalPropertyToColumnValueExtractor;
+
+/**
+ * This class converts DynamoDB Numbers to DECIMAL columns.
+ */
+public class DynamodbToDecimalPropertyToColumnValueExtractor
```

Comment:
Those names with two 'to' bother me...

Reply:
thats dificult since its a complex type hirachy:

ColumnMapping
PropertyToColumnMapping
AbstractPropertyToColumnMapping
ToDecimalPropertyToColumnMapping
ToStringPropertyToColumnMapping
ToJsonPropertyToColumnMapping

PropertyToColumnValueExtractor
AbstractToColumnValueExtractor
ToDecimalPropertyToColumnValueExtractor
 DynamodbToDecimalPropertyToColumnValueExtractor
...

The To expresses that decimal is the destination column.
The to in propertyToColumn that a DyanmoDB proerty is maped to a DynamoDB column.

Do you have a suggestion?

Reply:
I understand that the structure is complicated, but for me the names are unreadable now. I can't get what your classes are doing even after your comment. Can, for example, `ToDecimalPropertyToColumnMapping` be a `PropertyToDecimalColumnMapping`? Or does it do something different?

Reply:
wow, that good. Why did I not came top this ;-)